towncrier.toml

```diff
@@ -0,0 +1,50 @@
+[tool.towncrier]
+version = "2.15.0-dev0"
+directory = "doc/whatsnew/fragments"
+filename = "doc/whatsnew/2/2.15/index.rst"
+template = "doc/whatsnew/fragments/_template.rst"
+issue_format = "`#{issue} <https://github.com/PyCQA/pylint/issues/{issue}>`_"
```

Comment:
Open question : This look amazing, do you have an idea how we could create link like that for old changelog ? I've been wanting to create the link for some time now. Might be a one time sed now that we don't touch it anymore.

Reply:
Yes, I would probably go the same route - use `sed` or a simple helper script to replace all occurrences of issue numbers with a link.